workflows/dynamic_index.md

```diff
@@ -63,7 +72,13 @@ The arguments are as follows:
 12. **--consolidate_interval**: Granularity at which insert and delete functions are called.
 13. **--start_point_norm**: Set the starting node to a random point on a sphere of this radius.  A reasonable choice is to set this to the average norm of the data stream.
 
+** to build with filters add these
 
+14. **--label_file**: Filter data for each point, in `.txt` format. Line `i` of the file consists of a comma-separated list of filters corresponding to point `i` in the file passed via `--data_file`.
```

Comment:
list of labels instead of list of filters? Also, are labels numeric or can they be arbitrary strings?

Reply:
I have made the chnage in wording. And yes it does support any type of label. Both Strings and Numbers are supported as we create an internal map irrespective of the raw label type.